<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsSpatialQueryDialogBase</class>
 <widget class="QDialog" name="QgsSpatialQueryDialogBase">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>495</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Spatial Query</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="lytOperatorResultInvalid">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="lytOperator">
       <item>
        <widget class="QGroupBox" name="grpTargetGroupBox">
         <property name="toolTip">
          <string>Layer on which the topological operation will select geometries</string>
         </property>
         <property name="title">
          <string>Target layer</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QComboBox" name="cbTargetLayer">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Select the target layer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckbUsingSelectedTarget">
            <property name="toolTip">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt;&quot;&gt;When checked the operation will only consider selected geometries of the target layer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Selected feature(s) only</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>ckbUsingSelectedTarget</zorder>
         <zorder>cbTargetLayer</zorder>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="grpReferenceGroupBox">
         <property name="toolTip">
          <string>Layer whose geometries will be used as reference by the topological operation</string>
         </property>
         <property name="title">
          <string>Reference layer</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QComboBox" name="cbReferenceLayer">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Select the reference layer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckbUsingSelectedReference">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>18</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt;&quot;&gt;When checked the operation will be only consider selected geometries of the reference layer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Selected feature(s) only</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="grpOperationGroupBox">
         <property name="toolTip">
          <string>Topological operations between layers of target and reference</string>
         </property>
         <property name="title">
          <string>Topological operation</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QComboBox" name="cbOperantion">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Select the topological operation</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="lytResultInvalid">
       <item>
        <widget class="QTabWidget" name="twResultInvalid">
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabResult">
          <attribute name="title">
           <string>Result</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="lytTargetResult">
             <item>
              <widget class="QLabel" name="lbResultTarget">
               <property name="minimumSize">
                <size>
                 <width>18</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Target layer</string>
               </property>
               <property name="text">
                <string>Layer</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pbSelectResultTarget">
               <property name="toolTip">
                <string>New selection with list of FIDs</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qgsspatialquerydialogbase.qrc">
                 <normaloff>:/icons/select.png</normaloff>:/icons/select.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbSelectResultTargetAdd">
               <property name="toolTip">
                <string>Add selection with list of FIDs</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qgsspatialquerydialogbase.qrc">
                 <normaloff>:/icons/selectadd.png</normaloff>:/icons/selectadd.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbSelectResultTargetRemove">
               <property name="toolTip">
                <string>Remove selection with list of FIDs</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qgsspatialquerydialogbase.qrc">
                 <normaloff>:/icons/selectremove.png</normaloff>:/icons/selectremove.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="lbFIDresultTarget">
             <property name="toolTip">
              <string>List of FIDs from result of query</string>
             </property>
             <property name="text">
              <string>Feature IDs</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="lwResultFeatureTarget">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Select one FID to identify geometry of feature</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="lytSelected">
             <item>
              <widget class="QLabel" name="lbSelected">
               <property name="toolTip">
                <string>Number of selected features in map</string>
               </property>
               <property name="locale">
                <locale language="English" country="UnitedStates"/>
               </property>
               <property name="text">
                <string notr="true">selected features</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pbSelectedSubsetLayer">
               <property name="toolTip">
                <string>Set  subset in layer with list of FIDs</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qgsspatialquerydialogbase.qrc">
                 <normaloff>:/icons/selectesubsetlayer.png</normaloff>:/icons/selectesubsetlayer.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Invalid target</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="lytTargetInvalid">
             <item>
              <widget class="QLabel" name="lbInvalidTarget">
               <property name="toolTip">
                <string>Target layer</string>
               </property>
               <property name="text">
                <string>Layer</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pbSelectInvalidTarget">
               <property name="toolTip">
                <string>New selection with list of FIDs</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qgsspatialquerydialogbase.qrc">
                 <normaloff>:/icons/select.png</normaloff>:/icons/select.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="lbFIDinvalidTarget">
             <property name="text">
              <string>Feature IDs</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="lwInvalidFeatureTarget">
             <property name="toolTip">
              <string>Select item to identify geometry of feature</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Invalid reference</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="lytReferenceInvalid">
             <item>
              <widget class="QLabel" name="lbInvalidReference">
               <property name="toolTip">
                <string>Reference layer</string>
               </property>
               <property name="text">
                <string>Layer</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pbSelectInvalidReference">
               <property name="toolTip">
                <string>New selection with list of FIDs</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qgsspatialquerydialogbase.qrc">
                 <normaloff>:/icons/select.png</normaloff>:/icons/select.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="lbFIDinvalidReference">
             <property name="text">
              <string>Feature IDs</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="lwInvalidFeatureReference">
             <property name="toolTip">
              <string>Select item to identify geometry of feature</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="ckbZoomItem">
     <property name="text">
      <string>Zoom to identify FID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ckbLogProcessing">
     <property name="toolTip">
      <string>Check to show log processing of query</string>
     </property>
     <property name="text">
      <string>Show log messages</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="teStatus">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="lytProgressBarButtonsMain">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QProgressBar" name="pgbStatus">
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="bbMain">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Run query or close the window</string>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbTargetLayer</tabstop>
  <tabstop>ckbUsingSelectedTarget</tabstop>
  <tabstop>cbReferenceLayer</tabstop>
  <tabstop>ckbUsingSelectedReference</tabstop>
  <tabstop>cbOperantion</tabstop>
  <tabstop>bbMain</tabstop>
 </tabstops>
 <resources>
  <include location="qgsspatialquerydialogbase.qrc"/>
 </resources>
 <connections/>
</ui>
